src/overlays/actors/ovl_En_Blkobj/z_en_blkobj.c

```diff
@@ -77,13 +77,16 @@ void EnBlkobj_Destroy(Actor* thisx, PlayState* play) {
     DynaPoly_DeleteBgActor(play, &play->colCtx.dyna, this->dyna.bgId);
 }
 
+/**
+ * Draw Player's reflection until it is close enough to start the fight
+ */
 void EnBlkobj_Wait(EnBlkobj* this, PlayState* play) {
```

Comment:
feel like this comment makes it seem like this actor is doing the drawing of the reflection. dunno if we can change that phrasing to make it clear that its just telling link to do it

Reply:
"Wait for player to be in range to start the fight. Until then, signal that he should draw his own reflection", or something